dev/integration_tests/new_gallery/lib/feature_discovery/feature_discovery.dart

```diff
@@ -287,10 +287,14 @@ class _FeatureDiscoveryState extends State<FeatureDiscovery>
         setState(() {});
       })
       ..addStatusListener((AnimationStatus animationStatus) {
-        if (animationStatus == AnimationStatus.forward) {
-          setState(() => status = FeatureDiscoveryStatus.open);
-        } else if (animationStatus == AnimationStatus.completed) {
-          rippleController.forward(from: 0.0);
+        switch (animationStatus) {
+          case AnimationStatus.forward:
+            setState(() => status = FeatureDiscoveryStatus.open);
+          case AnimationStatus.completed:
+            rippleController.forward(from: 0.0);
+          case AnimationStatus.reverse:
+          case AnimationStatus.dismissed:
+            break;
```

Comment:
There isn't an `isForward` / `isReverse` getter, so I decided to do a switch statement here (for readability and to follow the [style guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-using-if-chains-or-or-with-enum-values)).